### PR TITLE
[S] Add tab-list component to UI library (#991)

### PR DESCRIPTION
**Pull request by tombyers**
Issue: https://github.com/repositive/report-issues/issues/991

### CODE REVIEW

File: tests/unit/components/r-tab-list-item/component-test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { setupComponentTest } from 'ember-mocha';
+import Ember from 'ember';
+
+const { get } = Ember;
+
+describe('Unit | Component | RUI / r-tab-list-item', function() {
+  setupComponentTest('rui/r-tab-list-item', {
+    needs: [
+      'validator:presence',
+      'validator:format'
```
Comment (tombyers) [inline tests/unit/components/r-tab-list-item/component-test.js]: @SebastianPlace These cause the tests to fail. I'm not sure where they are coming from, what dependency we need to include here.

File: tests/unit/components/r-tab-list/component-test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { describe, it, beforeEach } from 'mocha';
+import { setupComponentTest } from 'ember-mocha';
+import Ember from 'ember';
+
+const { get } = Ember;
+
+describe('Unit | Component | RUI / r-tab-list', function() {
+  setupComponentTest('rui/r-tab-list', {
+    needs: [
+      'validator:presence',
```
Comment (tombyers) [inline tests/unit/components/r-tab-list/component-test.js]: Also here.

**Comment by tombyers** [comment]
@SebastianPlace Should be ready to go

File: addon/components/r-tab-list-item/component.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  classNames: ['flex-auto cursor-pointer fc-secondary fs2 fw-med p2'],
+  classNameBindings: ['isActive:bc-white:bc-very-light-grey', 'isActive::u-hv-bc-very-light-grey'],
+
+  isActive: computed('tabName', 'activeTab', 'setActiveTab', function() {
+    if (get(this, 'setActiveTab')) {
+      return get(this, 'tabName') === get(this, 'activeTab');
+    } else {
+      const currentPath = get(this, 'router.currentPath');
+      const target = get(this, 'target');
+      return currentPath.includes(target);
```
Comment (SebastianPlace) [inline addon/components/r-tab-list-item/component.js]: I thought this was changed to `return currentPath.indexOf(target) > -1;`

Comment (tombyers) [reply addon/components/r-tab-list-item/component.js]: Well spotted. I changed this while the tabs were still in discover. And had already taken them across to the UI-library. Will fix

Comment (tombyers) [reply addon/components/r-tab-list-item/component.js]: @SebastianPlace Fixed

**Review by SebastianPlace — APPROVED** [review]
All good

**Pull request merged**